cogboard-webapp/src/components/widgets/types/PersonDrawWidget/index.js

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+
+import IconButton from '@material-ui/core/IconButton';
+import { TypographyVariant } from './styled';
+import { CenterWrapper } from '../TextWidget/styled';
+import { Refresh } from '@material-ui/icons';
+import { getIsAuthenticated } from '../../../../selectors';
+import { useSelector } from 'react-redux';
+import { array, bool, number } from 'prop-types';
+import { postWidgetContentUpdate } from '../../../../utils/fetch';
+
+const PersonDrawWidget = ({ id, multiTextInput, index }) => {
+  const isAuthenticated = useSelector(getIsAuthenticated);
+  const caption =
+    multiTextInput && multiTextInput.length - 1 >= index
+      ? multiTextInput[index]
+      : '';
+  const handleForceCycle = () => {
+    postWidgetContentUpdate({
+      id,
+      content: { forceCycle: true }
+    }).catch(e => console.log(e));
+  };
+
+  return (
+    <>
+      <CenterWrapper>
+        <TypographyVariant>{caption}</TypographyVariant>
```

Comment:
Do we really need this `TypographyVariant` ?

Reply:
fixed